cv32e40x/vendor_lib/verilab/README.md

```diff
@@ -0,0 +1,33 @@
+# Verilab SVLIB Vendor Library
+
+The Verilab SVLIB http://www.verilab.com/resources/svlib/ is a SystemVerilog package that provides a host of convenient utilities for
+testbench functionality that is not provided by UVM nor SystemVerilog itself conveniently.  These utilities include:
+
+- Advanced string handling
+- Regular expression parsing
+- File pathname manipulation
+- Environment variable queries
+- Time functions
+
+## Documentation
+
+For more details on using the SVLIB, please refer to their documentation PDF.  This PDF can be found either on the Verilab website
+or by cloning the Bitbucket SVLIB repository into vendor_lib (`make clone_svlib`) and finding the documenation under `svlib/doc`
+
+## CORE-V_VERIF COmpilation
```

Comment:
Typo: "COmpilation" s.b. "compilation".  (Wouldn't nag about something this simple, but I'm already asking for a change to `uvmt_cv32e40x_tb.sv`.)

Reply:
Fixed.  Push incoming.